src/main/java/io/debezium/connector/vitess/VitessSourceInfoStructMaker.java

```diff
@@ -21,9 +21,10 @@ public VitessSourceInfoStructMaker(
         super(connector, version, connectorConfig);
         this.schema = commonSchemaBuilder()
                 .name("io.debezium.connector.vitess.Source")
+                .field(SourceInfo.KEYSPACE_NAME_KEY, Schema.STRING_SCHEMA)
```

Comment:
"db" schema/value are set in the parent class; I already set keyspace value to it. For convenience of users, I added "keyspace" here.

Reply:
Idk if it is a good idea to put redundant info in the struct 🤷 
We can merge as is and see how it goes.

Reply:
I would call these database-specific info rather than redundant. Check out what other connectors do:

- https://github.com/debezium/debezium/blob/main/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresSourceInfoStructMaker.java
- https://github.com/debezium/debezium/blob/main/debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbSourceInfoStructMaker.java
- https://github.com/debezium/debezium/blob/main/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlSourceInfoStructMaker.java

Reply:
We are providing `keyspace` in multiple field in source: keyspace, db, schema - that's why I think there is some redundancy here.

Reply:
Yeah, the same value (keyspace) shouldn't be exported in multiple fields. Logged https://issues.redhat.com/browse/DBZ-4428 for sorting this out before the 1.8.0.Final due next week.